Add extra letter spacing modifier for inputs (#2230)
alphagov/govuk-frontend: CHANGELOG.md

This was added in [pull request #3187: Add disabled styles for form controls](https://github.com/alphagov/govuk-frontend/pull/3187).

### Added a modifier for text input styles that accept sequences of digits

> A new `.govuk-input--extra-letter-spacing` class for [Text Input](https://design-system.service.gov.uk/components/text-input/) was added to increase readability of text inputs meant to receive sequences of digits (like security codes, references or phone numbers).

Reviewer: ```suggestion
We've added a new `.govuk-input--extra-letter-spacing` class for [Text Input](https://design-system.service.gov.uk/components/text-input/). This increases readability of text inputs that receive sequences of digits (like security codes, references or phone numbers).
```
